transport.go

```diff
@@ -21,7 +33,8 @@ func (t *TailscaleCaddyTransport) UnmarshalCaddyfile(d *caddyfile.Dispenser) err
 func (t *TailscaleCaddyTransport) Provision(ctx caddy.Context) error {
 	t.logger = ctx.Logger()
 
-	s, err := getServer(ctx, "caddy-tsnet-client:80")
+	// TODO(will): allow users to specify a node name used to lookup that node's config in TSApp.
+	s, err := getNode(ctx, "caddy-tsnet-client:80")
```

Comment:
Should this be just `caddy-tsnet-client`?

Reply:
oh, good catch! I remember seeing that, and forgot to fix it